Features/sidebar

dipto0321 opened the pull request:
Hi @jklemon17 ,
We completed our sidebar to make it look like youtube.

dipto0321 committed:
initial markup and style for main

dipto0321 committed:
Add main images

rvvergara committed:
Style video title and social buttons below video

dipto0321 committed:
Finish vedio-info section

rvvergara committed:
Add images for sidebar and initial markup and style for sidebar

dipto0321 committed:
rename images for sideber

dipto0321 committed:
add video images list and add show more btn

rvvergara committed:
Fix sidebar widths and work on Show-More button styles

dipto0321 committed:
finish sidebar

dipto0321 committed:
add margin on body

jklemon17 commented:
I'm going to comment on this one, because it seems like the most complete of the pull requests. The site looks really good! :clap: I'm not an expert with CSS, so you may even be better than me here. A couple small comments:
- The horizontal lines next to the YouTube logo are lower than they should be.
- The search button is slightly higher than the search field
- The search field could use a little padding-left inside it, so the text doesn't start right at the edge.

Other than that, the only thing I'd say is in the future, be sure to make pull requests in order. I initially assumed the "nav" branch was the most complete, because it was the latest pull request, when in fact it was the least complete. Making pull requests as soon as you finish a feature should prevent this from happening in the future.

rvvergara commented:
Thanks @jklemon17

@dipto0321 in order to do the fixes suggested by Kyle here are my proposals:

1.  Fix hamburger button beside Youtube logo: Add (possibly at line 31 of style.css) the ff style:
    nav button {
      position: relative;
      bottom: 12px;
    } 

2. To fix padding on search box:
   - add padding: 3px 0 3px 10px to line 70 of style.css

3. To fix search button (@jklemon17 pls take note that if we go to the features/nav branch, the position of the search button isn't as high as this one in this branch but nonetheless yeah we can improve it ): 
 - Add position: relative and make top: 4px add to line 80 style.css
- change height in line 76 style.css from 26px to 30px (the size should increase considering we added padding to the search input bar in 2 above)
- Add border-radius of 2px (to make it look smooth).

rvvergara committed:
Fix hamburger menu, search-box and search button

1. Make nav button position relative and give it a bottom of 12px
2. Add padding of 3px 0px 3px 10px to .search input
3. Add position:relative and give it top: 4px, and change height from
26px to 30px (.search button)

rvvergara commented:
@jklemon17 and @dipto0321 Kindly check the latest commit for this branch as I implemented the above mentioned suggested fixes

dipto0321 committed:
Merge branch 'master' into features/sidebar
